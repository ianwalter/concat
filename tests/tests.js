const { test } = require('@ianwalter/bff')
const concat = require('..')

test('files get concatenated', async ({ expect }) => {
  const html = await concat(
    'tests/fixtures/book-*.html',
    'tests/fixtures/some-book.html',
    '**/book.html'
  )
  expect(html).toMatchSnapshot()
})
