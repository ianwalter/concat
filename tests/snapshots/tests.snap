// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`files get concatenated 1`] = `
"book one text
book two text
some book text
book text
"
`;
